examples/synology/Readme.md

```diff
@@ -0,0 +1,112 @@
+# Synology Installation
```

Comment:
For consistency within the repo, could you please adjust the case of this filename to `README.md`?